unetsocket/c/samples/rxdata.c

```diff
@@ -27,7 +27,7 @@ static int error(const char *msg) {
 int main(int argc, char *argv[]) {
     unetsocket_t sock;
     fjage_msg_t ntf;
-    uint8_t data[7];
+    uint8_t data[9];
```

Comment:
Can we make this `9` a #define?

Reply:
OK. Fixed in 7aa4cb925be204cda65dc607e8e25d43843f710d